### PR TITLE
Changes 'CONSULTATION NOTES' to 'GENERAL INSTRUCTIONS (ADVICE)'

### LINKED ISSUE #8213: Events module: Advise
**Describe the bug**
In consultation form there is a field to update "General Instructions (Advice)". When the user updates it, on events log the corresponding log calls it "CONSULTATION NOTES"

Change it in the events log to "General Instructions (Advice)"

**To Reproduce**
See this patient file: https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/14dd4345-450b-4392-9c5c-4f88bc32c533/consultation/952ebe29-8cdd-44fc-b13c-acde347f951b

### PR DESCRIPTION
**Pull request by ahmedobaid23**
## Proposed Changes

- Fixes #8213 

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by rithviknishad — COMMENTED** [review]
Nice attempt!

Although this works, there would be other `keys` that might need an altered display text eventually.

Consider adding a map or using translation files (preferred) to reverse the key to a display text.

For example, refer:

- https://github.com/coronasafe/care_fe/blob/develop/src/Components/Patient/ManagePatients.tsx#L1095
- https://github.com/coronasafe/care_fe/blob/develop/src/Locale/en/Common.json#L147-L150

**Comment by ahmedobaid23** [comment]
@rithviknishad 

Thanks for reviewing my PR.

I will create a new file named `ConsultationFormTranslation.json`. For now, I will only map `consultation_notes` to its relevant display text. In the future, if the display text of other keys needs to changed, they will be added and mapped to their corresponding display texts in this file. Sounds good?

**Comment by rithviknishad** [comment]
You can keep it in consultation.json itself. We have all consultation related translation keys present there.

File: src/Components/Facility/ConsultationDetails/Events/GenericEvent.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import type { ReactNode } from "react";
+import consultationMessage from "../../../../../src/Locale/en/Consultation.json";
 interface IProps {
   values: Record<string, unknown>;
 }
 
+type ConsultationMessageKeys = keyof typeof consultationMessage;
```
Comment (rithviknishad) [inline src/Components/Facility/ConsultationDetails/Events/GenericEvent.tsx]: You shouldn't be importing the translations file directly. Use `useTranslation` hook to retrieve the display text instead. Ref: https://github.com/coronasafe/care_fe/blob/f76248c997c1153a9f859511a6678ada18dd0af9/src/Components/Facility/Consultations/DailyRoundsList.tsx#L24

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by nihal467** [comment]
@ahmedobaid23 can you work on the changes requested

**Comment by ahmedobaid23** [comment]
@rithviknishad Yes, I am working on it

**Review by rithviknishad — APPROVED** [review]

**Review by shivankacker — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**